v2/src/features/UploadList/index.tsx

```diff
@@ -0,0 +1,3 @@
+import UploadButton from './UploadList';
```

Comment:
```suggestion
export ( default as UploadButton } from './UploadList';
```

Can be shorted to one line.